mod/metric_answer_lookup/lib/lookup_table.rb

```diff
@@ -4,7 +4,9 @@ def card
   end
 
   def refresh *fields
-    return delete if !card || card.trash || invalid?
+    # when we override a hybrid metric the answer is invalid because of the
+    # missing answer_id, so we check `invalid?` only for non-hybrid metrics)
```

Comment:
do I understand correctly that this means the answer_id is missing _before_ we override (but the answer is there once overridden?) Or are we not storing answer_ids when we override?

Reply:
the former is correct